client/src/components/Header.js

```diff
@@ -0,0 +1,43 @@
+import { ReactComponent as BurgerIcon } from "../icons/burger.svg";
+import styled from 'styled-components';
```

Comment:
```suggestion
import styled from 'styled-components/macro';
```

See: https://styled-components.com/docs/tooling#babel-macro